database/config.js

```diff
@@ -12,7 +12,7 @@ const config = {
   },
   test: {
     user: 'postgres',
-    password: '482106',
+    password: '123654',
```

Comment:
this seems to change all the time, can we figure out a way to all use the same password